### PR TITLE
Force builder images to Golang 1.22

### PR DESCRIPTION
**Pull request by dsimansk**
- Force builder images to Golang 1.22

/cc @pierDipi @mgencur

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -321,6 +321,7 @@ generate-dockerfiles:
 	rm -rf /tmp/serverless-operator-generator
 	$(shell go env GOPATH)/bin/generate \
 		--generators dockerfile \
+		--dockerfile-image-builder-fmt "registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17%.0s"  \
```
Comment (dsimansk) [inline Makefile:324]: `%s.0` is truncating string formatting to zero length, otherwise there's an extra param error in the result.

Comment (mgencur) [reply Makefile:324]: We could probably fix it somehow in openshift-knative/hack and allow passing just string in which case it would be used as is.

**Comment by dsimansk** [comment]
Update with a latest commit to `generate`.

**Review by pierDipi — APPROVED** [review]
/lgtm

**Comment by skonto** [comment]
/override "Red Hat Konflux / serverless-index-main-on-pull-request"

**Pull request merged**